app/views/reorder_timeline_entries/_reorder_list.html.erb

```diff
@@ -0,0 +1,17 @@
+<% timeline_entries = @coronavirus_page.timeline_entries.order(:position) %>
```

Comment:
I think this is good idea, but I think instead of doing it here, I'll create a card for it. That way the change can be applied to announcements and sub-sections at the same time.